### PR TITLE
Setup Presenter-Participant flow

### PR DESCRIPTION
**Pull request by VL4DYSL4V**

### CODE REVIEW

File: slidez-fe-core/package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "@fortawesome/free-solid-svg-icons": "^5.15.3",
     "@fortawesome/react-fontawesome": "^0.1.15",
     "@material-ui/core": "^4.12.3",
+    "@material-ui/icons": "^4.11.2",
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/package.json]: I don't think that we need it

File: slidez-fe-core/src/common/components/forms/LoginForm.tsx [hunk 2]
```diff
@@ -107,7 +107,7 @@ const LoginForm = ({ onLogin, onLoginWithGoogle }: LoginProps) => {
                                 name='email'
                                 className={
                                     'form-input' +
-                                    (viewErrors && (errors.email || loginError)
+                                    (viewErrors && errors.email
```
Comment (Anatolii-Ivanov) [inline slidez-fe-core/src/common/components/forms/LoginForm.tsx:114]: Natalia should fix it

Comment (VL4DYSL4V) [reply slidez-fe-core/src/common/components/forms/LoginForm.tsx:114]: If not removed, project won't build

File: slidez-fe-core/package.json (hunk 1 above)
Comment (VL4DYSL4V) [reply slidez-fe-core/package.json]: Removed

**Pull request merged**